src/components/NavigationMenu/NavigationMenu.scss

```diff
@@ -1,4 +1,4 @@
-.navigation-menu * {
+.navigation-menu ul {
```

Comment:
¿Queréis que **todos** los `ul` descendientes del `nav` tengan estos estilos?

Reply:
no, solo estos

Reply:
Ese selector dice "todos los `ul` descendientes del `nav`"

Reply:
sisi ya esta cambiado, le añadi una clase